src/Raven.Studio/typescript/common/shell/menu/items/settings.ts

```diff
@@ -133,7 +133,7 @@ function getSettingsMenuItem(appUrls: computedAppUrls) {
             route: 'databases/settings/revertRevisions',
             moduleId: bridgeToReact(RevertRevisions, "nonShardedView"),
             shardingMode: "allShards",
-            title: 'Revert Revisions',
+            title: 'Revert Collections',
```

Comment:
you are renaming a feature that is called like that in numerous places incl Client API, etc. This will cause a lot of inconsistency.

Reply:
@ppekrol 

**What do you suggest that should be done ?**

The issue is that we need to differentiate between reverting entire Collections (as a whole)
-vs- reverting specific documents to specific revisions (the new operation).

As far as I know, there is no Client API Operation to do the first option, you can only do that via the Studio.
Only the second option has a Client API operation which I am documenting.

"Revert revisions" isn't a clear terminology, as it’s Not the revisions being reverted but rather the documents (within collections) reverted to older revisions.
"Revert collections" is actually what is being done, and more accurately describes the action, 
and this matches the UI presented to the user (selecting a collection and a point in time to revert to).

With the new operation added,
users will have a choice to either "revert a collection" or "revert a specific document"

Reply:
@ppekrol 

How about this idea:

Rename just the label/title of this view to "Revert Documents to Revisions"
That would cover both cases:
* the current UI (where you select Collections)
* and the new UI option that will be added in [RavenDB- 22956](https://issues.hibernatingrhinos.com/issue/RavenDB-22956/Expose-an-option-to-revert-current-document-version-to-a-previous-revision) where you can select a specific document to revert.